keyboards/enviousdesign/commissions/mini1800/info.json

```
@@ -0,0 +1,110 @@
+                { "matrix": [1, 9], "x": 9.5, "y": 1 },
+                { "matrix": [1, 10], "x": 10.5, "y": 1 },
+                { "matrix": [1, 11], "x": 11.5, "y": 1 },
+                { "matrix": [1, 12], "w": 1.5, "x": 12.5, "y": 1 },
+                { "matrix": [1, 14], "x": 14.5, "y": 1 },
+                { "matrix": [1, 15], "x": 15.5, "y": 1 },
+                { "matrix": [1, 16], "x": 16.5, "y": 1 },
+                { "matrix": [2, 0], "w": 1.75, "x": 0, "y": 2 },
+                { "matrix": [2, 1], "x": 1.75, "y": 2 },
+                { "matrix": [2, 2], "x": 2.75, "y": 2 },
+                { "matrix": [2, 3], "x": 3.75, "y": 2 },
+                { "matrix": [2, 4], "x": 4.75, "y": 2 },
+                { "matrix": [2, 5], "x": 5.75, "y": 2 },
+                { "matrix": [2, 6], "x": 6.75, "y": 2 },
+                { "matrix": [2, 7], "x": 7.75, "y": 2 },
+                { "matrix": [2, 8], "x": 8.75, "y": 2 },
+                { "matrix": [2, 9], "x": 9.75, "y": 2 },
+                { "matrix": [2, 10], "x": 10.75, "y": 2 },
+                { "matrix": [2, 11], "w": 1.75, "x": 11.75, "y": 2 },
+                { "matrix": [2, 14], "x": 14.5, "y": 2 },
+                { "matrix": [2, 15], "x": 15.5, "y": 2 },
+                { "matrix": [2, 16], "x": 16.5, "y": 2 },
+                { "matrix": [3, 0], "w": 2, "x": 0, "y": 3 },
+                { "matrix": [3, 1], "x": 2, "y": 3 },
+                { "matrix": [3, 2], "x": 3, "y": 3 },
+                { "matrix": [3, 3], "x": 4, "y": 3 },
+                { "matrix": [3, 4], "x": 5, "y": 3 },
+                { "matrix": [3, 5], "x": 6, "y": 3 },
+                { "matrix": [3, 6], "x": 7, "y": 3 },
+                { "matrix": [3, 7], "x": 8, "y": 3 },
+                { "matrix": [3, 8], "x": 9, "y": 3 },
+                { "matrix": [3, 9], "x": 10, "y": 3 },
+                { "matrix": [3, 11], "w": 1.75, "x": 11, "y": 3 },
+                { "matrix": [3, 14], "x": 14.5, "y": 3 },
+                { "matrix": [3, 15], "x": 15.5, "y": 3 },
+                { "matrix": [3, 16], "x": 16.5, "y": 3 },
+                { "matrix": [3, 12], "x": 13, "y": 3.25 },
+                { "matrix": [4, 0], "w": 1.25, "x": 0, "y": 4 },
+                { "matrix": [4, 1], "w": 1.25, "x": 1.25, "y": 4 },
+                { "matrix": [4, 2], "w": 1.25, "x": 2.5, "y": 4 },
+                { "matrix": [4, 4], "w": 2.75, "x": 3.75, "y": 4 },
+                { "matrix": [4, 6], "w": 2.75, "x": 6.5, "y": 4 },
+                { "matrix": [4, 8], "w": 1.25, "x": 9.25, "y": 4 },
+                { "matrix": [4, 10], "w": 1.25, "x": 10.5, "y": 4 },
+                { "matrix": [4, 15], "x": 15.5, "y": 4 },
+                { "matrix": [4, 16], "x": 16.5, "y": 4 },
+                { "matrix": [4, 11], "x": 12, "y": 4.25 },
+                { "matrix": [4, 12], "x": 13, "y": 4.25 },
+                { "matrix": [4, 13], "x": 14, "y": 4.25 }
```

Normalizes the key sequence. (1/3)

```suggestion
                { "matrix": [3, 11], "w": 1.75, "x": 11, "y": 3 },
                { "matrix": [3, 12], "x": 13, "y": 3.25 },
                { "matrix": [3, 14], "x": 14.5, "y": 3 },
                { "matrix": [3, 15], "x": 15.5, "y": 3 },
                { "matrix": [3, 16], "x": 16.5, "y": 3 },
                { "matrix": [4, 0], "w": 1.25, "x": 0, "y": 4 },
                { "matrix": [4, 1], "w": 1.25, "x": 1.25, "y": 4 },
                { "matrix": [4, 2], "w": 1.25, "x": 2.5, "y": 4 },
                { "matrix": [4, 4], "w": 2.75, "x": 3.75, "y": 4 },
                { "matrix": [4, 6], "w": 2.75, "x": 6.5, "y": 4 },
                { "matrix": [4, 8], "w": 1.25, "x": 9.25, "y": 4 },
                { "matrix": [4, 10], "w": 1.25, "x": 10.5, "y": 4 },
                { "matrix": [4, 11], "x": 12, "y": 4.25 },
                { "matrix": [4, 12], "x": 13, "y": 4.25 },
                { "matrix": [4, 13], "x": 14, "y": 4.25 },
                { "matrix": [4, 15], "x": 15.5, "y": 4 },
                { "matrix": [4, 16], "x": 16.5, "y": 4 }
```